### DISCUSSION TITLE
当游戏用户使用用户名作为账户名登录时，后续操作只返回这个用户名对应的档案

### DISCUSSION THREAD

**Discussion by Silverteal** [bs-community/blessing-skin-plugins#238] [Ideas]
由于Mojang API的行为中，一次请求只返回一个用户档案，有一些应用（如 Geyser）会依赖这一行为，从而导致 BSS 的多用户支持并不能最好发挥作用。如果当游戏用户使用用户名作为账户名登录时，后续操作只返回这个用户名对应的档案信息，则可能有助于缓解这一问题。

**Comment by tnqzh123** [comment]
Implemented in `yggdrasil-connect` plugin.
